autoupdate/main.go

```diff
@@ -241,15 +241,32 @@ func update(log *zap.Logger) int {
 
 	// start btfs function test
 	if ready_to_test {
-		if get_functest(log, btfsBinaryPath) {
-			log.Info("BTFS daemon get file test succeeded!")
-		} else {
-			log.Error("BTFS daemon get file test failed!")
+		test_success := false
+		// try up to two times
+		for i := 0; i < 2; i++ {
+			if get_functest(log, btfsBinaryPath) {
+				log.Info("BTFS daemon get file test succeeded!")
+				test_success = true
+				break
+			} else {
+				log.Error("BTFS daemon get file test failed!")
+			}
 		}
-		if add_functest(log, btfsBinaryPath) {
-			log.Info("BTFS daemon add file test succeeded!")
-		} else {
-			log.Error("BTFS daemon add file test failed!")
+		if !test_success {
+			os.Exit(101)
```

Comment:
Can we define the error code instead of being a random number here?

Reply:
i had felt it's optional to name a variable that's only used once.  but i agree it helps readability.  Jin is making a change on this file in his PR and i have requested him to put in the change while he is there.